src/DataProtection/Azure.KeyVault.Keys/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- These projects depend on a 3rd party source. -->
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
This setting should do nothing. Only ref/ projects and those in the Shared FX are built in source build.